vivisect/symboliks/common.py

```diff
@@ -81,6 +81,7 @@ def __init__(self):
         self.kids          = []
         self.parents       = []
         self.cache         = {}
+        self._reduced       = False
```

Comment:
yea, i dig the idea of changing over to a while todo kinda thing as long as we can ensure depth first processing.